README.md

```diff
@@ -3,6 +3,9 @@ Riot
 
 Riot (formerly known as Vector) is a Matrix web client built using the [Matrix React SDK](https://github.com/matrix-org/matrix-react-sdk).
 
+Riot is officially supported on the web in modern versions of Chrome, Firefox, and Safari. Other browsers may work, however
```

Comment:
Should we clarify that this means the desktop versions of those browsers only, so people don't assume mobile browsers as well? (Is there a way to say that without the antiquated word "desktop"...?)

Reply:
I ended up adding words for "please use mobile apps" instead. How does it sound?